array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -10,33 +10,35 @@
 #SBATCH --output=DNAm/logFiles/%u/QCDNAdata.log # output file
 #SBATCH --job-name=QCDNAdata
 
+#------------------------------------------------------
+
+# 1. the first input is your project folder.
+# 2. the second input is the full path to config file.
+# 3. the third input is a the path to the config.r file.
+
+#-----------------------------------------------------
```

Comment:
Thanks for putting this at the top of the page. It can be quite difficult for the user sometimes to know what the inputs to the script does. This makes it much more transparent.